frontend/src/metabase/lib/dom.js

```diff
@@ -13,6 +13,13 @@ export const getScrollY = () =>
 // Cypress renders the whole app within an iframe, but we want to exlude it from this check to avoid certain components (like Nav bar) not rendering
 export const isWithinIframe = function () {
   try {
+    if (
+      isCypressActive &&
+      window.location.search.includes("enableCypressIframe=true")
```

Comment:
Why not test the change using real preview in the app and not with a fake parameter?